variable: tidb_enable_ts_validation (#22758)
pingcap/docs: system-variables.md

</CustomContent>

### `tidb_enable_ts_validation` <span class="version-mark">New in v9.0.0</span>

- Scope: GLOBAL
- Persists to cluster: Yes

> - Controlled by the Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No

Author: ```suggestion
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
```

- Type: Boolean
- Default value: `ON`

> - This variable controls whether TiDB validates the legality of the timestamp in a request before sending the request to TiKV. If an illegal timestamp is detected in the request, TiDB rejects sending the request to avoid data corruption.

Author: ```suggestion
- This variable controls whether TiDB validates the timestamp in a request before sending the request to TiKV. If an invalid timestamp is detected in the request, TiDB rejects sending it to avoid data corruption.
```
